test/jdk/java/beans/XMLEncoder/Test4625418.java

```diff
@@ -23,7 +23,7 @@
 
 /*
  * @test
- * @bug 4625418 8239965
+ * @bug 4625418 8239965 8290488
```

Comment:
I'm curious why this XMLEncoder test is updated, is this related to round trip?

Reply:
Yes. The test assumes all code points below U+10000 have 1-1 roundtrip (c2b2c). Since the assumption is now not met with this change, I commented out the 864 entries.